src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -0,0 +1,444 @@
+          onTouchTap: this._onMenuItemTouchTap.bind(this, index, child.props.value),
+        }, child.props.children);
+        index += 1;
+        return clone;
+      });
+
+    let popoverStyle;
+    if (anchorEl && !autoWidth) {
+      popoverStyle = {width: anchorEl.clientWidth};
+    }
+
+    return (
+      <div
+        {...other}
+        ref="root"
+        className={className}
+        style={styleUtils.prepareStyles(muiTheme, styles.root, open && styles.rootWhenOpen, style)}
+      >
+        <ClearFix style={styleUtils.merge(styles.control)} onTouchTap={this._onControlTouchTap}>
+          <div style={styleUtils.prepareStyles(muiTheme, styles.label, open && styles.labelWhenOpen, labelStyle)}>
+            {displayValue}
+          </div>
+          <DropDownArrow style={styleUtils.merge(styles.icon, iconStyle)}/>
+          <div style={styleUtils.prepareStyles(muiTheme, styles.underline, underlineStyle)}/>
+        </ClearFix>
+        <Popover
+          anchorOrigin={{horizontal: 'left', vertical: 'top'}}
+          anchorEl={anchorEl}
+          style={popoverStyle}
+          animation={PopoverAnimationFromTop}
+          open={open}
+          onRequestClose={this._onMenuRequestClose} >
+          <Menu
+            maxHeight={maxHeight}
+            desktop={true}
+            value={value}
+            style={menuStyle}
+            >
+            {menuItemElements}
+          </Menu>
+        </Popover>
+      </div>
+    );
+  },
+
+  _testDeprecations() {
+    warning(this.props.displayMember === 'text',
+      'displayMember will be removed in favor of composability. refer to the documentation for more information');
+
+    warning(this.props.labelMember === 'text',
+      'labelMember will be removed in favor of composability. refer to the documentation for more information');
+
+    warning(!this.props.hasOwnProperty('menuItems'),
+      'menuItems will be removed in favor of composability. refer to the documentation for more information');
+
+    warning(!this.props.hasOwnProperty('selectedIndex'),
+      'selectedIndex will be removed. use value instead to control the component.');
+
+    warning(!this.props.hasOwnProperty('valueLink'),
+      'valueLink will be removed. use value and onChange');
+
+    warning(this.props.valueMember === 'payload',
+      'valueMember will be removed in favor of composability. refer to the documentation for more information');
+  },
+
+  _setWidth() {
+    const el = ReactDOM.findDOMNode(this);
+    if (!this.props.style || !this.props.style.hasOwnProperty('width')) {
+      el.style.width = 'auto';
+    }
+  },
+
+  _setSelectedIndex(index) {
+    warning(index >= 0, 'Cannot set selectedIndex to a negative index.');
+    this.setState({selectedIndex: (index >= 0) ? index : 0});
+  },
+
+  _onControlTouchTap(event) {
+    event.preventDefault();
+    if (!this.props.disabled) {
+      this.setState({
+        open: !this.state.open,
+        anchorEl: ReactDOM.findDOMNode(this),
```

Comment:
Same here.

I tested and it looks like it's working properly still but maybe give it once over and make sure I'm right.